cmake/VecCoreConfig.cmake.in

```diff
@@ -3,22 +3,22 @@ set(VecCore_VERSION_MAJOR @VecCore_VERSION_MAJOR@)
 set(VecCore_VERSION_MINOR @VecCore_VERSION_MINOR@)
 set(VecCore_VERSION_PATCH @VecCore_VERSION_PATCH@)
 
-get_filename_component(VecCore_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+@PACKAGE_INIT@
 
-list(APPEND CMAKE_MODULE_PATH "${VecCore_CMAKE_DIR}")
+set_and_check(VecCore_INCLUDE_DIR "@PACKAGE_CMAKE_INSTALL_INCLUDEDIR@")
+set_and_check(VecCore_CMAKE_DIR "${CMAKE_CURRENT_LIST_DIR}")
 
-set(VecCore_ROOT_DIR "${VecCore_CMAKE_DIR}/../../..")
-set(VecCore_INCLUDE_DIR  "${VecCore_ROOT_DIR}/include")
-set(VecCore_INCLUDE_DIRS "${VecCore_ROOT_DIR}/include")
+set(VecCore_INCLUDE_DIRS "${VecCore_INCLUDE_DIR}")
+list(APPEND CMAKE_MODULE_PATH "${VecCore_CMAKE_DIR}")
 
 set(_VecCore_FIND_QUIET)
-if (VecCore_FIND_QUIETLY)
+if(VecCore_FIND_QUIETLY)
```

Comment:
Could you please remove the style changes like these? I prefer to keep using the current style.